### PR TITLE
Inline NoClip Permited

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -381,13 +381,7 @@ public bool IsOverwatchEnabled
...
         {
             get => FpcNoclip.IsPermitted(ReferenceHub);
-            set
-            {
-                if (value && !FpcNoclip.PermittedPlayers.Contains(ReferenceHub.netId))
-                    FpcNoclip.PermitPlayer(ReferenceHub);
-                else if (!value && FpcNoclip.PermittedPlayers.Contains(ReferenceHub.netId))
-                    FpcNoclip.UnpermitPlayer(ReferenceHub);
-            }
+            set => _ = value ? FpcNoclip.PermitPlayer(ReferenceHub) : FpcNoclip.UnpermitPlayer(ReferenceHub)
```
Comment (iopietro) [inline Exiled.API/Features/Player.cs:384]:
```suggestion
            set => value ? FpcNoclip.PermitPlayer(ReferenceHub) : FpcNoclip.UnpermitPlayer(ReferenceHub)
```

**Review by iopietro — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (IRacle1) [reply Exiled.API/Features/Player.cs:384]: It wont work compile without `_ =`, C# operators moment

Comment (iopietro) [reply Exiled.API/Features/Player.cs:384]: Yes, because you can't assign a void value to the discard operator `_`.

**Comment by Thundermaker300** [comment]
Does this *need* to be inline? I think it's fine as is

**Comment by IRacle1** [comment]
> Does this _need_ to be inline? I think it's fine as is

Yeah, i dont see anything sus in it

**Comment by iopietro** [comment]
It cannot be done inline, so no.

**Comment by louis1706** [comment]
> It cannot be done inline, so no.

for me is work pefectly fine it's just need to use _ = who are equal to use OpCodes.Pop in Transpiler

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.API/Features/Player.cs:384]: the problem is from a missing ;

**Review by Thundermaker300 — APPROVED** [review]

**Review by iopietro — APPROVED** [review]

**Pull request merged**